llvm/test/CodeGen/PowerPC/ppc-shrink-wrapping.ll

```diff
@@ -463,7 +463,7 @@ for.preheader:
 
 for.body:                                         ; preds = %entry, %for.body
   %i.03 = phi i32 [ %inc, %for.body ], [ 0, %for.preheader ]
-  tail call void asm "addi 14, 14, 1", "~{r14}"()
+  tail call void asm "addi 31, 14, 1", "~{r31}"()
```

Comment:
nit: Should we change the second 14 to 31 too? That's still GPR 14.

Reply:
The second r14 is an input, so I think it is Ok to keep it the same.

Reply:
OK, I see. PEI only inserts save/restore for non-volatile CSRs which have defs.

I agree, we can keep the `r14`